test/handlers/compile-tests.ts

```diff
@@ -42,19 +41,21 @@ const languages = {
 
 describe('Compiler tests', () => {
     let app, compileHandler;
-
     beforeAll(() => {
         const compilationEnvironment = makeCompilationEnvironment({languages});
         compileHandler = new CompileHandler(compilationEnvironment, fakeProps({}));
 
-        const textParser = bodyParser.text({type: () => true});
-        const formParser = bodyParser.urlencoded({extended: false});
-
         app = express();
-        app.use(bodyParser.json());
-
-        app.post('/noscript/compile', formParser, compileHandler.handle.bind(compileHandler));
-        app.post('/:compiler/compile', textParser, compileHandler.handle.bind(compileHandler));
+        app.use(express.json());
+        app.use(express.text({type: () => true}));
+        app.use(express.urlencoded({extended: false}));
```

Comment:
Can you explain why these moved here? This appears to be a change in behaviour, but perhaps this is reinstating some defaults that were lost from `body-parser` ? These also appear to be duplicated in the lines below? Please let us know why this was needed. Thanks!